website/versioned_docs/version-0.62/releases.md

```diff
@@ -0,0 +1,25 @@
+---
+id: version-0.62-releases
+title: Release Strategy (vnext)
+original_id: releases
+---
+
+> This page is hanging, see [Issue #59](https://github.com/microsoft/react-native-windows-samples/issues/59).
+
+This document describes the release strategy for the [`vnext`](https://github.com/microsoft/react-native-windows/blob/master/README.md) effort for React Native for Windows.
```

Comment:
Remove vnext. This should just be release strategy for React Native for Windows.